lib/queue.js

```diff
@@ -346,6 +346,10 @@ function redisOptsFromUrl(urlString) {
       }
     }
 
+    if (redisUrl.protocol && redisUrl.protocol.startsWith('rediss')) {
+      redisOpts.tls = {};
+    }
```

Comment:
Good change. Default should be to use TLS for `rediss` protocol, and you can still disable that with query string options like `tls=false`.